go-controller/pkg/metrics/cluster_manager.go

```diff
@@ -54,6 +54,30 @@ var metricV6AllocatedHostSubnetCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The total number of v6 host subnets currently allocated",
 })
 
+/** EgressIP metrics recorded from cluster-manager begins**/
+var metricEgressIPCount = prometheus.NewGauge(prometheus.GaugeOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemMaster,
```

Comment:
sub system needs to be changed to the new component.

Reply:
ah good catch! done.